packages/core/shared/call-api.ts

```diff
@@ -34,7 +34,10 @@ export async function callApi({
       messages,
       ...body,
     }),
-    headers,
+    headers: {
+      'Content-Type': 'application/json',
+      ...headers,
+    },
```

Comment:
Makes sense!

Reply:
fyi, this caused a breaking change in my app since my server was expecting `text/plain`